File: dotnetWebApi/Data/Entities/Tasinmaz.cs
namespace Data.Entities
{
    public class Tasinmaz
    {
        public int id { get; set; }

        public int IlId { get; set; }
        public  Mahalle Mahalle{ get; set; }
        // public Il Il { get; set; }

        public int IlceId { get; set; }
        // public Ilce Ilce { get; set; }

        public int MahalleId { get; set; }
        
        
        public string Adres { get; set; }

        public string AppUserId { get; set; }
        public AppUser AppUser { get; set; }

        public int Parsel { get; set; }
        public int Ada { get; set; }
        public string Nitelik { get; set; }
        public string  XCoordinate { get; set; }
        public string  YCoordinate { get; set; }
        public string ParselCoordinate  { get; set; }

        
    }
}

File: webAuth/src/app/Models/tasinmaz.ts
export class Tasinmaz
{
  id:number;
  ilId:number;
  ilceId:number;
  mahalleId:number;
  adres:string;
  parsel:number;
  ada:number;
  nitelik:string;
  xCoordinate:string;
  yCoordinate:string;
  parselCoordinate:string;
  //appUserId:string;

  constructor(id = 0, ilId=0,ilceId=0,mahalleId=0,adres='',parsel=0,ada=0,nitelik='',xCoordinate='',yCoordinate='',parselCoordinate='')
  {
    this.id = id;
    this.ilId = ilId;
    this.ilceId = ilceId;
    this.mahalleId = mahalleId;
    this.adres = adres;
    this.parsel = parsel;
    this.ada = ada;
    this.nitelik = nitelik;
    this.xCoordinate = xCoordinate;
    this.yCoordinate = yCoordinate;
    this.parselCoordinate = parselCoordinate;
  }


}
